addUser.php

```diff
@@ -192,9 +192,11 @@ class="radio">
     <script type="application/json" json-data id="mailSettings">
 <?php
       $mail_templates = array();
-      foreach (MAIL_TEMPLATES as $name => $template) {
-        $mail_templates[] = array('name' => $name,
-          'template' => file_get_contents(BASE_PATH . $template));
+      foreach (MAIL_TEMPLATES['addUser'] as $template) {
+        $mail_templates[] = array(
+		  'name' => $template['name'],
+		  'subject' => $template['subject'],
```

Comment:
$mailSettings['subject'] (below) should not be needed anymore (and MAIL_SUBJECT is not defined anymore)

Reply:
Not handled yet